### PR TITLE
[Merged by Bors] - feat: LinearEquivClass QuadraticForm.IsometryEquiv

### PR DESCRIPTION
**Pull request by alexjbest**
From the lorentz center workshop https://github.com/alexjbest/ant-lorentz

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/LinearAlgebra/QuadraticForm/IsometryEquiv.lean [hunk 1]
```diff
@@ -96,6 +96,10 @@ def trans (f : Q₁.IsometryEquiv Q₂) (g : Q₂.IsometryEquiv Q₃) : Q₁.Iso
     map_app' := by intro m; rw [← f.map_app, ← g.map_app]; rfl }
 #align quadratic_form.isometry.trans QuadraticForm.IsometryEquiv.trans
 
+instance : LinearEquivClass (Q₁.IsometryEquiv Q₂) R M₁ M₂ where
+  map_add := fun f a b => map_add f.toLinearMap a b
+  map_smulₛₗ := fun f a b => map_smulₛₗ f.toLinearMap a b
+
```
Comment (eric-wieser) [inline Mathlib/LinearAlgebra/QuadraticForm/IsometryEquiv.lean:102]: Can you move this right below the `EquivLike` instance? Or can it even replace it entirely?

**Comment by eric-wieser** [comment]
bors d+

**Comment by alexjbest** [comment]
thanks!

**Pull request closed without merging**